consensus/fork.go

```diff
@@ -98,6 +98,7 @@ func (s *State) generateAndApplyDiff(bn *blockNode) (err error) {
 	s.currentBlockID = bn.block.ID()
 	s.currentPath[bn.height] = bn.block.ID()
 
+	// TODO: minerSubsidy isn't used anywhere?
```

Comment:
no, what happened is there's a new mechanism for dealing with the miner fees and payouts. The validity of the payouts is checked earlier in the code (when the block is first accepted, before calling fork), thus making this code unnecessary.

I cleaned it out on my branch don't worry about it here.